kvariants/src/lib.rs

```diff
@@ -34,15 +33,15 @@ pub static KVARIANTS: Lazy<HashMap<char, KVariant>> = Lazy::new(|| {
     //   㓻 (U+34FB)	sem	    剛 (U+525B)
     //   ...
     //
-    let tsv = include_str!("../../../../dictionaries/txt/chinese/kVariants.tsv");
+    let dictionary = include_str!("../dictionaries/compressed/kVariants.csv");
```

Comment:
Then we retrieve the dictionary directly from the out dir.
```suggestion
    let dictionary = include_bytes!(concat!(env!("OUT_DIR"), "/kVariants.csv"));
```

Reply:
Also fixed in 2c326cd.